native/packages/react-native-bpk-component-input/src/BpkInput.js

```diff
@@ -34,32 +34,44 @@ const tokens = Platform.select({
   android: () => ANDROID_TOKENS,
 })();
 
+const {
+  colorGray100,
```

Comment:
We should have actualy tokens specific for textinput. e.g `textInputBorderColor` rather than `colorGray100`